x-pack/plugins/infra/server/lib/alerting/metric_threshold/register_metric_threshold_alert_type.ts

```diff
@@ -37,9 +37,13 @@ const FIRED_ACTIONS = {
 };
 
 const getCurrentValueFromAggregations = (aggregations: Aggregation) => {
-  const { buckets } = aggregations.aggregatedIntervals;
-  const { value } = buckets[buckets.length - 1].aggregatedValue;
-  return value;
+  try {
```

Comment:
Out of curiosity, why the try/catch here. Why not just explicit `if (!buckets.length)`? Not a huge deal, but I'm afraid that intent is lost with try/catch.

Reply:
Fair enough, figured we wanted to guard for any error in retrieving the data but I'll see if I can clarify